make test error in gravmag.tensor.center_of_mass

There is a small error which results in a failed make test.  There appears to just be an extra space in the returned array which results in the arrays not being treated as equal.

======================================================================
FAIL: Doctest: fatiando.gravmag.tensor.center_of_mass
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/Users/mkass/anaconda/lib/python2.7/doctest.py", line 2226, in runTest
    raise self.failureException(self.format_failure(new.getvalue()))
AssertionError: Failed doctest test for fatiando.gravmag.tensor.center_of_mass
  File "/Users/mkass/Projects/fatiando/fatiando/gravmag/tensor.py", line 140, in center_of_mass

----------------------------------------------------------------------
File "/Users/mkass/Projects/fatiando/fatiando/gravmag/tensor.py", line 189, in fatiando.gravmag.tensor.center_of_mass
Failed example:
    cm
Expected:
    array([-100.,  20., 100.])
Got:
    array([-100.,   20.,  100.])


----------------------------------------------------------------------

Fatiando version: 
>>> fatiando.__version__
'0.3-329-gc477695-dirty'

--- a/doc/changelog.rst
+++ b/doc/changelog.rst
@@ -10,6 +10,8 @@ Version (development)
 
 **Changes**:
 
+* Fix the doctest for ``fatiando.gravmag.tensor.center_of_mass``.
+  (`PR 242 <https://github.com/fatiando/fatiando/pull/242>`__)
 * **BUG FIX**: Tesseroid computations failed (silently) when tesseroids were
   smaller than 1e-6 degrees on a side (~ 10 cm). Code now ignores these
   tesseroids on input and warns the user about it. If a tesseroid becomes

--- a/fatiando/gravmag/tensor.py
+++ b/fatiando/gravmag/tensor.py
@@ -186,8 +186,8 @@ def center_of_mass(x, y, z, eigvec1, windows=1, wcenter=None, wmin=None,
     >>> eigenvals, eigenvecs = tensor.eigen(data)
     >>> # Now estimate the center of mass
     >>> cm = tensor.center_of_mass(x, y, z, eigenvecs[0])
-    >>> cm
-    array([-100.,  20., 100.])
+    >>> print('{:.1f} {:.1f} {:.1f}'.format(cm[0], cm[1], cm[2]))
+    -100.0 20.0 100.0
 
     """
     if wmin is None:

